src/analyses/mCP.ml

```diff
@@ -65,6 +65,12 @@ struct
     let deps (x,_) = iter (check_dep x) @@ (find_spec x).dep in
     iter deps xs
 
+  let specific_cont_sens xs =
+    (*most analysis must be set to context insensitive, because we only want to analyse context sensitively for a specific analysis*)
```

Comment:
```suggestion
    (* most analyses must be set to context insensitive, because we only want to analyse context sensitively for a specific analysis *)
```